packages/paste-website/src/components/site-wrapper/ScrollAnchorIntoView.tsx

```diff
@@ -1,6 +1,8 @@
 import * as React from 'react';
 import {getCurrentPathHash} from '../../utils/RouteUtils';
 
+const Y_OFFSET = -90;
```

Comment:
We do have this partly in a constant for the site assuming this is an offset to accomodate for the masthead https://github.com/twilio-labs/paste/blob/main/packages/paste-website/src/constants.ts#L5

Obviously doesn't include padding to make it look nice